ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -302,7 +303,7 @@ def _create_session_core(
             )
 
         session_info = session_infos[0]
-        with closing_session(session_constructor(session_info)) as session:
+        with closing_session(session_constructor(session_info), initialization_behavior) as session:
```

Comment:
Doesn't the initialization parameter also need to be bubbled out through the "public" APIs for creating sessions?

Reply:
It is not in the generic API.

```
    @requires_feature(SESSION_MANAGEMENT_2024Q1)
    def create_session(
        self,
        session_constructor: Callable[[SessionInformation], TSession],
        instrument_type_id: str,
    ) -> ContextManager[TypedSessionInformation[TSession]]:
```

Reply:
I've added them now.